src/main/java/com/auctionapp/api/controller/AuctionController.java

```diff
@@ -34,4 +36,11 @@ public AuctionController(final AuctionService auctionService) {
 
 		return ResponseEntity.status(HttpStatus.OK).body(auctions);
 	}
+
+	@GetMapping("/{id}")
+	public ResponseEntity<AuctionDto> get(@PathVariable final UUID id) {
+		AuctionDto result = service.getAuction(id);
```

Comment:
~~ can also be final. Generally, anything that is not reassigned for some reason within a method can and should be final.